source/app.d

```diff
@@ -51,13 +52,22 @@ void main(string[] args)
 	if (!CLI!DlingsCmdlArgs.parseArgs(cmdlArgs, args[1 .. $]))
 		return;
 
+
+	if (!exists("exercises.toml"))
+	{
+		writeln("file 'exercises.toml' is missing!");
+		return;
```

Comment:
You should rather exit with an erroneous error code

Reply:
True, I should fix all other instances too, I will fix this in a new case.

Reply:
Filed an issue: #12